src/Sarif.Converters.UnitTests/TSLintLoaderTests.cs

```diff
@@ -20,7 +18,7 @@ public void TSLintLoader_ReadLog_WhenInputIsNull_ThrowsArgumentNullException()
         {
             TSLintLoader loader = new TSLintLoader();
 
-            Action action = () => loader.ReadLog(null);
+            Action action = () => loader.ReadLog(default(Stream));
```

Comment:
@michaelcfanning The change isn't just for readability. Now that there are overloads `ReadLog(string)` and `ReadLog(Stream)`, you can't just pass `null` because it's ambiguous.